packages/scratch-vm/src/extensions/scratch3_musiccreation/sheetmusic.js

```diff
@@ -626,6 +643,7 @@ class SheetMusic {
         if (x > 120 && y < 0) {
             vizHelper.clearSheetMusicList();
         }
+        this.setPenColorToColor(this.black,util);
```

Comment:
this one probably isn't necessary, just for safety

Reply:
Ya, definitely looks like it. The logic at [Line 594](https://github.com/mitmedialab/prg-extension-boilerplate/pull/122/files#diff-96a475ff727047b5d1864dbc6974a931455dfa6ab6f902803c3d63a09dc15db8R594) should handle it all